nursery/encode-data-using-base64-in-dotnet.yml

```diff
@@ -0,0 +1,12 @@
+rule:
+  meta:
+    name: encode data using Base64 in .NET
+    namespace: data-manipulation/encoding/base64
```

Comment:
can we combine with https://github.com/mandiant/capa-rules/blob/master/data-manipulation/encoding/base64/encode-data-using-base64.yml?

Reply:
Sure, https://github.com/mandiant/capa-rules/pull/579/commits/cc712c5c6f083ed7a1ecfb7f5a5414e289363d33 performs the merge with the rule.